src/deadline_worker_agent/installer/install.sh

```diff
@@ -203,6 +204,13 @@ else
     fi
     set -e
 fi
+if [[ "${python_interpreter_path}" == "unset" ]]; then
+    echo "ERROR: --python-interpreter-path is not specified"
+    usage
+elif [[ ! -f "${python_interpreter_path}" ]]; then
+    echo "ERROR: The Python interpreter path is not found: \"${python_interpreter_path}\""
+    usage
+fi
```

Comment:
Can we default this to whichever python shows up first in the search path to make it non-required?

Reply:
We want to use the same Python interpreter in which the worker agent program files are installed - especially when the worker agent is installed in a Python virtual environment. Otherwise the worker agent's dependencies may not be available or may have conflicting versions.

Again, because this is always provided by the `install-deadline-worker` entrypoint ([source](https://github.com/jusiskin/deadline-cloud-worker-agent/blob/cda9398085fffb15641a757e60dde20070551be1/src/deadline_worker_agent/installer/__init__.py#L125-L126)), this only provides a guardrail from calling this bash script from a different source which is not something the worker agent officially supports.